packages/react-native-app-auth/ios/RNAppAuth.m

```diff
@@ -733,8 +791,13 @@ - (NSString*)getErrorMessage: (NSError*) error {
     }
 }
 
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
 - (id<OIDExternalUserAgent>)getExternalUserAgentWithPresentingViewController: (UIViewController *)presentingViewController
                                                      prefersEphemeralSession: (BOOL *) prefersEphemeralSession
+#elif TARGET_OS_OSX
+- (id<OIDExternalUserAgent>)getExternalUserAgentWithPresentingWindow: (NSWindow *)presentingWindow
+                                             prefersEphemeralSession: (BOOL *) prefersEphemeralSession
+#endif
```

Comment:
Note that this `(BOOL *)` should actually be written as `(BOOL)`. We should change it to `(BOOL)` after merging my PR https://github.com/FormidableLabs/react-native-app-auth/pull/1000.